### ISSUE TITLE
Sharding

### ISSUE THREAD

**Issue by zubairfazal** [Androz2091/AtlantaBot#137]
**Is your feature request related to a problem? Please describe.**
no
**Describe the solution you'd like**
Implement sharding to the bot 

**Describe alternatives you've considered**
N/A

**Additional context**
Would like to have sharding/cluster in the bot

**Comment by zubairfazal** [comment]
Sorry not a bug, but a suggestion

**Comment by zubairfazal** [comment]
Sorry its a suggestion not a bug

**Comment by zubairfazal** [comment]
Any luck?

**Comment by Androz2091** [comment]
Maybe this will be added but I don't think so since the original bot only has ~680 servers.

**Comment by Priyanshu348** [comment]
But u should give the code with sharding

**Comment by ezywebca** [comment]
i agree with @Androz2091 as the bot hasnt even capped the 2500 server mark this is a matter to more or less worry around 2000 discord servers

**Comment by Developer-Doge** [comment]
I think this would be a nice feature because some people would like to see how something like this would be implemented in a already existing bot like this one.
